crates/aranya-fast-channels/src/shm/align.rs

```diff
@@ -12,12 +12,18 @@ use super::shared::assert_ffi_safe;
 /// Reports whether `v` is aligned to `align`.
 pub(super) fn is_aligned_to(v: usize, align: usize) -> bool {
     assert!(align.is_power_of_two());
-    v % align == 0
+    #[allow(clippy::arithmetic_side_effects)] // checking alignment
```

Comment:
`// checking alignment` is a bad comment. The real reason is that `align` is (must be) a power of two and is therefore non-zero. Also, use `#[allow(..., reason = "...")]` to explain why.